### DISCUSSION TITLE
Supabase support

### DISCUSSION THREAD

**Discussion by jameschetwood** [keystonejs/keystone#8821] [Questions] [answered]
I've tried to connect a new Keystone install to an existing Supabase DB. I already have a table called "Profiles" which I created in Supabase. When I connect Keystone I get this warning:

```
⚠️  Warnings:
  • You are about to drop the `Profiles` table, which is not empty (3 rows).
```

Does this mean Supabase isn't a compatable DB?

**Comment by dcousens** [comment, marked as answer]
If you are using `keystone dev` with an existing database, the [`db push`](https://www.prisma.io/docs/reference/api-reference/command-reference#db-push) functionality may result in data loss as a result of any schema changes.

Your databases appears to be compatible.
